llama_hub/tools/zapier/base.py

```diff
@@ -39,20 +53,21 @@ def list_actions(self):
         return response.text
 
 
-    def natural_language_query(self, id: str, instructions: str):
+    def natural_language_query(self, id: str, params: Dict[str, str]):
         """
         Make a natural language action to Zapier
         This endpoint accepts natural language instructions to integrate with other services
+        You should always provide a natural language string in the params dict descrbing the overall action to be taken
 
+        The action being called must have an id obtained from list_actions. If the action is not exposed it can be here: 
         Args:
             id (str): The id of the zapier action to call
-            instructions (str): The natural language instruction to pass to Zapier
-
+            params (Optional[dict]): The instructions and other values instructing the action to be taken
         """
 
         response = requests.post(
```

Comment:
for future iterations maybe we could think about error handling (if we can find a failure case)